lib/entities/app-definition.ts

```diff
@@ -68,9 +76,14 @@ export type AppDefinitionProps = {
   /**
    * Locations where the app can be installed
    */
-  locations: LocationDefinition[]
+  locations: Location[]
```

Comment:
Do you mind renaming this type to be a bit more specific and indicating that this is actually the location of an app?